Assignment_05.rb

```diff
@@ -95,22 +138,25 @@ def initialize
   end
 
   def add_one_task(task_attributes)
-    task = Task.new(task_attributes[:id], task_attributes[:name], task_attributes[:time_logs])
-    if task.valid?
-      @tasks << task
-    else
-      puts "Task with id: #{task_attributes[:id]} not added"
+    begin
+      task = Task.new(task_attributes[:id], task_attributes[:name], task_attributes[:time_logs])
+      if task.valid?
+        @tasks << task
+      else
+        puts "Task with id: #{task_attributes[:id]} not added"
+      end
+    rescue => e
+      puts "Error occured in adding task: #{e.message}"
     end
   end
 
   def add_many_task(task_attr_array)
```

Comment:
@AritraChatterjee-qi Can you please rename this method as `add_tasks` instead of `add_many_task`